internal/models/factor.go

```diff
@@ -141,7 +141,7 @@ func NewFactor(user *User, friendlyName string, factorType string, state FactorS
 // FindFactorsByUser returns all factors belonging to a user ordered by timestamp
 func FindFactorsByUser(tx *storage.Connection, user *User) ([]*Factor, error) {
 	factors := []*Factor{}
-	if err := tx.Q().Where("user_id = ?", user.ID).Order("created_at asc").All(&factors); err != nil {
+	if err := tx.Eager().Q().Where("user_id = ?", user.ID).Order("created_at asc").All(&factors); err != nil {
```

Comment:
Primarily used here: https://github.com/supabase/gotrue/pull/1371/files#diff-1333bda583e387c5dd0bd9638211f7981cf6491c037096634102ec7c64cca745L93

We regularly delete challenges in row cleanup so shouldn't incur too much performance overhead